.github/workflows/base_node_binaries.yml

```
@@ -269,17 +269,20 @@ jobs:
         # S3OPTIONS: '--recursive --exclude "*" --include "*.zip*"'
         # S3OPTIONS: '--acl public-read'
 
-  create-release:
-    runs-on: ubuntu-latest
-    needs: builds
-    if: ${{ env.AWS_SECRET_ACCESS_KEY != '' && matrix.os != 'self-hosted' && startsWith(github.ref, 'refs/tags/v') }}
-    steps:
-      - name: Download binaries
-        uses: actions/download-artifact@v2
-      - name: Create release
-        uses: ncipollo/release-action@v1
-        with:
-          artifacts: "tari_*/**/*"
-          token: ${{ secrets.GITHUB_TOKEN }}
-          prerelease: true
-          draft: true
+# "The workflow is not valid. .github/workflows/base_node_binaries.yml (Line: 273, Col: 9): Unrecognized named-value: 'env'. Located at position 1 within expression: env.AWS_SECRET_ACCESS_KEY != '' ...
+  # create-release:
+  #   runs-on: ubuntu-latest
+  #   needs: builds
+  #   if: ${{ env.AWS_SECRET_ACCESS_KEY != '' && matrix.os != 'self-hosted' && startsWith(github.ref, 'refs/tags/v') }}
```

Should be this. The workflow is valid otherwise, but I want it to only run on tags

```suggestion
  #   if: ${{ startsWith(github.ref, 'refs/tags/v') }}
```

---

Please uncomment the section and apply this change. Otherwise I'm happy to merge

---

Sure, I only commented it out since it broke the workflow

---

done as 2 commits